ts/Dashboard/SerializeHelper/HTMLTableConverterHelper.ts

```diff
@@ -0,0 +1,107 @@
+/* *
+ *
+ *  (c) 2020 - 2021 Highsoft AS
+ *
+ *  License: www.highcharts.com/license
+ *
+ *  !!!!!!! SOURCE GETS TRANSPILED BY TYPESCRIPT. EDIT TS FILE ONLY. !!!!!!!
+ *
+ *  Authors:
+ *  - Pawel Lysy
+ *  - Sophie Bremer
+ *
+ * */
+
+'use strict';
+/* *
+ *
+ *  Import
+ *
+ * */
+
+import HTMLTableConverter from '../../Data/Converters/HTMLTableConverter';
+import Serializable from '../Serializable.js';
+import U from '../../Core/Utilities.js';
+const { merge } = U;
+
+/* *
+ *
+ *  Functions
+ *
+ * */
+/**
+ * Change the obj of HTMLTableConverter to its Serialized form.
+ * @param obj Object to serialize
+ * @return Serialized object
+ */
+
```

Comment:
Unnecessary space, while at the top one is missing.

Reply:
Space, the final frontier... should go to line 16.